patterns/multisig/VirtualMultisigRune.ts

```diff
@@ -172,7 +172,10 @@ export class VirtualMultisigRune<out C extends Chain, out U>
     const proxiesGrouped = Rune
       .tuple([proxies, membersCount])
       .map(([proxies, membersCount]) =>
-        [proxies.slice(0, membersCount), proxies[membersCount]] as [Uint8Array[], Uint8Array]
+        [
+          proxies.slice(0, membersCount),
+          stash ?? proxies[membersCount],
```

Comment:
Stash could be a Rune resolving to undefined

Reply:
Good catch. Fixed